### DISCUSSION TITLE
How to add zh_core_web_sm tokenizer?

### DISCUSSION THREAD

**Discussion by jet10000** [code-kern-ai/refinery#58] [General]
https://newreleases.io/project/github/explosion/spacy-models/release/zh_core_web_sm-3.2.0

<img width="483" alt="image" src="https://user-images.githubusercontent.com/2258120/180918224-47d8759a-d5d2-456e-b8dd-03fd8060375e.png">

**Comment by jhoetter** [comment]
Hi @jet10000! Thanks for trying _refinery_ :)

That seems to be missing in the docs, we'll add it. To do add `zh_core_web_sm`, you need to go to the config page (click on your avatar on the top right of your screen). There you can pick the available tokenizers:
<img width="1440" alt="config_screen" src="https://user-images.githubusercontent.com/62521561/180951571-acfb335b-963c-4a9d-8df0-258c0b0f0c2f.png">

Once you add `zh_core_web_sm`, it will be shown in your project creation screen as an option:
<img width="1440" alt="new_tokenizer" src="https://user-images.githubusercontent.com/62521561/180951577-b4b9cda2-7067-4be2-8d48-de8de2c5baa5.png">

Hope that helps!
